inc/dashboard/widget.class.php

```diff
@@ -233,6 +294,13 @@ class="card big-number"
    }
 
 
+   public static function summaryNumber(array $params = []): string {
+      $params['class'] = 'summary-numbers';
+      //$params['label'] = '';
```

Comment:
```suggestion
```

Reply:
it can be an array (see docblock), i think mixed type is not available with our php version requirement

Reply:
the fact is:
- when we have an empty value , we receive ""
- when we have a filled value, we receive ["date1", "date2]

Reply:
heu... it's just about removing a commented line here :)

Reply:
wrong place, it was for the previous entry, sorry

Reply:
np, so `$values` is either an array or an empty string, therefore whecking its lenght is not required

Reply:
yes, commit is coming